src/ImageNodeSpec.js

```diff
@@ -33,7 +34,9 @@ function getAttrs(dom: HTMLElement) {
       ps.width &&
       ps.height &&
```

Comment:
could the `width` and `height` run into a similar issue?

Reply:
seems unlike, cause user will never make an image with zero width or height.